addons/mail/static/src/owl/components/composer_text_input/composer_text_input.js

```diff
@@ -27,553 +17,105 @@ class ComposerTextInput extends Component {
-         */
-        this.MENTION_THROTTLE = 200;
         this.storeDispatch = useDispatch();
         this.storeGetters = useGetters();
-        this.storeProps = useStore(state => {
+        this.storeProps = useStore((state, props) => {
             return {
                 isMobile: state.isMobile,
+                composer: state.composers[props.composerLocalId],
             };
         });
         /**
-         * jQuery node reference to textarea that is used to configure
-         * summernote editor on. Useful to invoke summernote commands.
-         */
-        this._$textarea = undefined;
-        /**
-         * Node reference of the contenteditable that is part of summernote
-         * editor. Useful to interact with content of the composer.
-         */
-        this._editable = undefined;
-        /**
-         * Last tracked range in the contenteditable. Useful to correctly insert
-         * some content at the correct position.
-         */
-        this._lastRange = undefined;
-        /**
-         * Reference of the textarea. Only useful to compute the jQuery node
-         * reference of it...
+         * Reference of the textarea. Useful to set height, selection and content.
          */
         this._textareaRef = useRef('textarea');
-        /**
-         * Tribute instance that has been configured on contenteditable of
-         * summernote editor. Tribute is a small framework to enable mentions
-         * in the contenteditable.
-         */
-        this._tribute = undefined; // list of tribute mentions (partner, canned responses, etc.)
-
-        this._searchChannelMentionSuggestions = _.throttle(
-            this._searchChannelMentionSuggestions.bind(this),
-            this.MENTION_THROTTLE
-        );
-        this._searchPartnerMentionSuggestions = _.throttle(
-            this._searchPartnerMentionSuggestions.bind(this),
-            this.MENTION_THROTTLE
-        );
     }
 
     /**
-     * Load summernote, since this is not in the assets bundle.
+     * Updates the composer text input content when composer has changed
+     * as textarea content can't be changed from the DOM.
      */
-    willStart() {
-        return this._loadSummernote();
-    }
-
-    /**
-     * Configure summernote and tribute on mount
-     */
-    mounted() {
-        const {
-            $textarea,
-            editable,
-        } = this._configSummernote();
-        const tribute = this._configTribute({ editable });
-
-        this._$textarea = $textarea;
-        this._editable = editable;
-        this._tribute = tribute;
-
-        if (this.props.initialHtmlContent) {
-            this.setHtmlContent(this.props.initialHtmlContent);
-        }
-        this._update(); // remove initial <p></br></p>
-    }
-
-    willUnmount() {
-        this._tribute.detach(this._editable);
-        this._$textarea.summernote('destroy');
+    patched() {
+        this._textareaRef.el.value = this.storeProps.composer.textInputContent;
```

Comment:
already handled by template with `t-esc` on `<textarea>`

Reply:
```t-es``` allows to set the default value of a textarea only. However, to set this value again (when changing tab for example), we have to specifically set the value of the textarea otherwise the value has only been updated in the store, not in the view.